.vib/mongodb/goss/mongodb.yaml

```
@@ -0,0 +1,17 @@
+file:
+  /opt/bitnami/mongodb/mongodb_get_source.txt:
+    mode: "0644"
+    exists: true
+    contains:
+      - This solution ships the official MongoDB Community binaries
```

Minor comment, I think checking for such a long string is very error-prone if upstream decides to change some of it.

```suggestion
      - official MongoDB Community
```